packages/terra-dropdown-button/docs/DropdownButton.md

```diff
@@ -0,0 +1,18 @@
+# Terra Dropdown Button
+
+The Terra Dropdown Button is a button that, when pressed, opens a dropdown with more buttons.
+It is intended to allow the user to choose from a list of options with no special emphasis on any option.
+
+Children must be `Button` subcomponent for proper functionality and appearance.
```

Comment:
This seems a little ambiguous between the terra-button and the DropdownButton's Button.